CMakeSettings.json

```diff
@@ -0,0 +1,61 @@
+{
+    // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
+    "configurations": [
+      {
+        "name": "x86-Debug",
+        "generator": "Visual Studio 15 2017",
+        "configurationType": "Debug",
+        "buildRoot": "${env.LOCALAPPDATA}\\CMakeBuild\\${workspaceHash}\\build\\${name}",
+        "cmakeCommandArgs": "",
+        "buildCommandArgs": "-m -v:minimal",
+        "variables": [
+          {
+            "name": "LUG_ROOT",
+            "value": "C:\\Lugdunum3D"
```

Comment:
I don't think it's a good idea to put your own path in a file which would be for everyone.